lib/dialects/mssql/query-generator.js

```diff
@@ -815,6 +815,11 @@ const QueryGenerator = {
       mainFragment += ' AS ' + mainTableAs;
     }
 
+    const tableHints = ['NOLOCK', 'READUNCOMMITTED', 'UPDLOCK', 'REPEATABLEREAD', 'SERIALIZABLE', 'READCOMMITTED', 'TABLOCK', 'TABLOCKX', 'PAGLOCK', 'ROWLOCK', 'NOWAIT', 'READPAST', 'XLOCK', 'SNAPSHOT', 'NOEXPAND'];
```

Comment:
Since we are hard coding the available values anyway, we might as well expose it as an object, similar to datatypes. That way users can also get editor completion

https://github.com/mccabemj/sequelize/blob/ca4d995f80d5ca22563061adb3ea51aed0852094/lib/sequelize.js#L1165
https://github.com/mccabemj/sequelize/blob/ca4d995f80d5ca22563061adb3ea51aed0852094/lib/query-types.js

Reply:
Thanks @janmeier good suggestion. I have implemented that, let me know what you think